uber/site_sections/registration.py

```diff
@@ -24,20 +24,22 @@ def check_everything(attendee):
     if message:
         return message
 
-    if AT_THE_CON and not attendee.age_group and attendee.is_new:
+    if AT_THE_CON and not COLLECT_EXACT_BIRTHDATE and not attendee.age_group and attendee.is_new:
         return "You must enter this attendee's age group"
 
 
 @all_renderable(PEOPLE, REG_AT_CON)
 class Root:
-    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first'):
-        total_count = session.query(Attendee).count()
+    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+        filter = Attendee.status.in_([NEW_STATUS, COMPLETED_STATUS, PRINTED_STATUS]) if not invalid else None
```

Comment:
On second thought, I'm going to use a new column for printing instead, since it isn't a 'status' in the same sense these other statuses are.